specification/network/resource-manager/readme.md

```diff
@@ -60,6 +60,7 @@ input-file:
 - Microsoft.Network/stable/2018-06-01/virtualNetworkGateway.json
 - Microsoft.Network/stable/2018-06-01/vmssNetworkInterface.json
 - Microsoft.Network/stable/2018-06-01/vmssPublicIpAddress.json
+- Microsoft.Network/stable/2018-06-01/virtualWan.json

```

Comment:
should we sort this alphabetically? @jianghaolu, please confirm if so.

Reply:
I don't think it's necessary though if it seems more organized to service team then it would be nice to have.